Features.md

```diff
@@ -13,24 +13,46 @@
    * Profile Photo
    * Email (for authentication related)
    * Hobbies
-   * NUmber (for authentication related)
+   * Number (for authentication related)
    * Kind of places love to visit
    * Current location
-   * User Travel Histroy (the places the user has been to)
+
 * User calendar log : 
-   * Wheter the status of user is free or not on a particular date
-* User tarveling log : 
-   * If going out :
-      * Place going to visit
-      * Is he alone or with a group
-      * Need a group to visit
-      * Invites to be sent (finds friends to sent the Invite)
+   * Current_Status
+   * Current_Date
```

Comment:
In place of current date it would be better if you use a specific date, whether or not Current date ? what do u think @RajuKoushik